sdk/digitaltwins/azure-digitaltwins-core/README.md

```diff
@@ -18,7 +18,7 @@ For the best development experience, developers should use the official Microsof
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-digitaltwins-core</artifactId>
-  <version>1.0.0-beta.2</version>
```

Comment:
I am reaching out to the SDK team to see if there is an automated approach to update the version in the readme file.

Reply:
This version currently doesn't automatically gets updated with every release we do.